src/main/java/com/alrex/parcool/utilities/MathUtil.java

```diff
@@ -16,4 +21,22 @@ public static float normalizeRadian(float angle) {
     public static float normalizeDegree(float angle) {
         return (float) (angle - 360f * Math.floor((angle + 180f) / 360f));
     }
+	
+	public static Vec3 rotateYDegrees(Vec3 vector, float baseAngle)
+	{
+		var angle = baseAngle * Mth.DEG_TO_RAD;
+		return new Vec3(vector.x * Mth.cos(angle) - vector.z * Mth.sin(angle), vector.y, vector.x * Mth.sin(angle) + vector.z * Mth.cos(angle));
+	}
+
+	public static float toYaw(Vec3 vector) {
+		return (float)Math.toDegrees(Math.atan2(-vector.x, vector.z));
+	}
+
+	public static boolean isZero(Vec3 vector) {
+		return vector.x == 0 && vector.y == 0 && vector.z == 0;
+	}
+
+	public static boolean isZero(Vec2 vector) {
+		return vector.x == 0 && vector.y == 0;
+	}
```

Comment:
Which should this be in `MathUtil`, or `VectorUtil`? Or should I integrate them?